lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -300,6 +303,17 @@ def _format_network_info(self, address):
             'selfLink': address
         }
 
+    def _format_metadata(self, metadata):
+        '''
+            :param metadata: A list of dicts where each dict has keys "key" and "value"
+            :return a dict with key/value pairs for each in list.
+        '''
+        new_metadata = {}
+        print(metadata)
```

Comment:
DO NOT PRINT, if you need to display something use display class, in any case there is no reason for an inventory plugin to do any printing into screen.